src/main/java/com/box/sdk/Metadata.java

```diff
@@ -133,10 +136,29 @@ public String get(String path) {
         if (value == null) {
             return null;
         }
+        if (value.isNumber()) {
+            return value.toString();
+        }
         return value.asString();
     }
 
     /**
+     * Returns a list of metadata proeprty paths.
+     * @return the list of metdata property paths.
+     */
+    public List<String> getKeys() {
```

Comment:
Should this be renamed, maybe `getPropertyPaths`? I wouldn't expect it to return JSON patch paths from the cur func name